### PR TITLE
favorite_category table created

### PR DESCRIPTION
**Pull request by jonathan-portillo**
Got rid of leftover tag table and replaced it with the favorite_category table.  
Favorite_category currently has a relationship with buyer_profile and is only looking for categories to be added into the table.

### CODE REVIEW

**Review by maxwes13 — DISMISSED** [review]

File: data/migrations/20201109234734_seller_profie_and_item.js [hunk 1]
```diff
@@ -20,9 +20,15 @@ exports.up = function (knex) {
       tb.increments();
       tb.string('category_name', 255);
     })
-    .createTable('tag', (tb) => {
+    .createTable('favorite_category', (tb) => {
       tb.increments();
-      tb.string('tag_name', 255);
+      tb.string('categories', 255);
```
Comment (Brodt7258) [inline data/migrations/20201109234734_seller_profie_and_item.js]: I think we need to reference the categories table here

**Review by Brodt7258 — CHANGES_REQUESTED** [review]

File: data/migrations/20201109234734_seller_profie_and_item.js (hunk 1 above)
Comment (jonathan-portillo) [reply data/migrations/20201109234734_seller_profie_and_item.js]: Got it updated!!

File: data/migrations/20201109234734_seller_profie_and_item.js [hunk 2]
```diff
@@ -20,9 +20,19 @@ exports.up = function (knex) {
       tb.increments();
       tb.string('category_name', 255);
     })
-    .createTable('tag', (tb) => {
+    .createTable('favorite_category', (tb) => {
       tb.increments();
-      tb.string('tag_name', 255);
+      tb.string('categories', 255)
```
Comment (Brodt7258) [inline data/migrations/20201109234734_seller_profie_and_item.js]: sorry, should have noted this in my first comment, but can you switch that to the category id?

**Review by Brodt7258 — CHANGES_REQUESTED** [review]

File: data/migrations/20201109234734_seller_profie_and_item.js (hunk 2 above)
Comment (jonathan-portillo) [reply data/migrations/20201109234734_seller_profie_and_item.js]: fixed!

**Review by Brodt7258 — APPROVED** [review]
thanks for bearing with me, looks great now.

**Review by maxwes13 — APPROVED** [review]

**Pull request merged**